samples/nrf_rpc/entropy_nrf53/cpuapp/CMakeLists.txt

```diff
@@ -5,12 +5,8 @@
 #
 cmake_minimum_required(VERSION 3.13.1)
 
-set(NRF_SUPPORTED_BOARDS
-  nrf5340pdk_nrf5340_cpuapp
-)
-
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
-project(entropy_nrf53_cpuapp)
+project(NONE)
```

Comment:
Still nice to have a `project(<some-name>)` but can be done later.

Reply:
Will fix later. Don't want to re-trigger the CI.